system/doc/getting_started/seq_prog.xml

```diff
@@ -737,12 +736,12 @@ list_max([Head|Rest], Result_so_far)  ->
       the next part of the function.</p>
     <p>Some useful operators in guards are, &lt; less than, &gt;
       greater than, == equal, &gt;= greater or equal, =&lt; less or
-      equal, /= not equal. (see the chapter
-      <seealso marker="doc/reference_manual:expressions">"Guard Sequences"</seealso> in the Erlang Reference Manual).</p>
+      equal, /= not equal. (See the chapter
+      <seealso marker="doc/reference_manual:expressions">"Guard Sequences"</seealso> in the Erlang Reference Manual.)</p>
```

Comment:
It's not uncommon in standard usage and is grammatically correct. For more examples of it within these docs, see @59ce273. There examples of it in grammar references [here](http://grammar.ccc.commnet.edu/grammar/marks/parentheses.htm), [here](http://blog.apastyle.org/apastyle/2013/03/punctuation-junction-periods-and-parentheses.html), and [here](http://www.grammarbook.com/punctuation/parens.asp).

We could always remove the parentheses in this case, if they seem unnecessary.